lib/assets/SnapshotAlertManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SnapshotAlertManager.swift
+//
+//  Created by Maxime Britto on 06/01/2016.
+//  Copyright © 2016 dev2a. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+var alertInterruptionMonitor:NSObjectProtocol?
+
+func automaticallyDismissAlerts(testCase:XCTestCase) -> NSObjectProtocol
+{
+    return SnapshotAlertManager.automaticallyDismissAlerts(testCase)
+}
+
+func stopDismissingAlerts(testCase:XCTestCase) {
+    SnapshotAlertManager.stopDismissingAlerts(testCase)
+}
+
+func waitForAlertsToBeDismissed() {
+    SnapshotAlertManager.waitForAlertsToBeDismissed()
+}
+
```

Comment:
I'm just a bit curious why these are global functions. Should/could they be class functions of SnapshotAlertManager?